### DISCUSSION TITLE
Navigate between `<NavigationContainers />`

### DISCUSSION THREAD

**Discussion by thebiltheory** [react-navigation/react-navigation#9802] [Q&A]
Hi there, 

Did anyone come across a case where one had to navigate between `<NavigationContainer>`'s?

Let me explain: 
I have a main `NavigationContainer` which handles all my `Navigators` (`Stack'`).
However, I started using [react-native-bottom-sheet](https://github.com/gorhom/react-native-bottom-sheet).
I have another `NavigationContainer` when the `bottom-sheet` opens.

The structure looks as follows: 
```
- App
-- <NavigationContainer> Number 1
---- <StackNavigator>
----- <StackScreens>
----- <StackScreen with Bottom Sheet>
------ <NavigationContainer independent={true}> Number 2 
------- <StackNavigator>
```

What I'm trying to achieve here is to go from one of the screens inside NavigationContainer number 2 to a screen inside NavigationContainer 1.

Example of [Bottom Sheet Implementation](https://github.com/gorhom/react-native-bottom-sheet/blob/master/example/src/screens/integrations/NavigatorExample.tsx) with react-navigation (5 I believe)

**Comment by ReevMich** [comment]
yep im stuck with this as well. Did you find a solution?

**Comment by satya164** [comment]
The whole point of `independent={true}` is to have a completely independent container and prevent navigation between them. If you don't want to prevent navigation then why are you using multiple `NavigationContainer`s and `independent={true}`?
